indra/explanation/model_checker/pybel.py

```diff
@@ -27,14 +27,20 @@ def __init__(self, model, statements=None, do_sampling=False, seed=None):
         super().__init__(model, statements, do_sampling, seed)
         self.model_agents = self._get_model_agents()
 
-    def get_graph(self):
+    def get_graph(self, include_variants=False, symmetric_variant_links=False,
+                  include_components=True, symmetric_component_links=True):
```

Comment:
By default hasComponent edges are included in both directions and hasVariant edges are not included at all in PybelModelChecker